### PR TITLE
Update documentation for v6

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: .doctrine-project.json [hunk 1]
```diff
@@ -5,21 +5,27 @@
...
             "branchName": "master",
             "slug": "latest",
             "upcoming": true
         },
         {
-            "name": "5.0",
-            "branchName": "5.x",
-            "slug": "5.0",
+            "name": "6.0",
+            "branchName": "6.x",
```
Comment (carusogabriel) [inline .doctrine-project.json]: I noticed that we don't have the `6.x` branch. Should we create it?

Comment (Ocramius) [reply .doctrine-project.json]: Only after release, not yet.

**Comment by Ocramius** [comment]
Note: to be merged after release 😛

**Comment by Ocramius** [comment]
Ouff, just noticed that this works already with 6.x, not 7.x. It should've been merged earlier: sorry, @carusogabriel!

**Pull request merged**